lib/datasource/docker/types.ts

```diff
@@ -9,6 +9,7 @@ export enum MediaType {
   manifestV1 = 'application/vnd.docker.distribution.manifest.v1+json',
   manifestV2 = 'application/vnd.docker.distribution.manifest.v2+json',
   manifestListV2 = 'application/vnd.docker.distribution.manifest.list.v2+json',
+  ociManifestV1 = 'application/vnd.oci.image.manifest.v1+json',
```

Comment:
Should we also add support for `application/vnd.oci.image.index.v1+json`[^1]? The OCI equivalent of docker's manifest list

[^1]: https://github.com/opencontainers/image-spec/blob/main/image-index.md

Reply:
Sounds good